Userland/Libraries/LibGfx/ImageFormats/WebPLoaderLossyTables.h

```diff
@@ -11,6 +11,214 @@
 namespace Gfx {
 
 using Prob = u8;
+using tree_index = i8;
+
+// https://datatracker.ietf.org/doc/html/rfc6386#section-10 "Segment-Based Feature Adjustments"
+const tree_index mb_segment_tree[2 * (4 - 1)] = {
```

Comment:
Hrm. These snake_case names violate our coding standard, even though it's the exact structure/enum names laid out in the spec.

Reply:
Sounding like please change the nice spec-copied names to be PascalCase to better match the rest of the project style :). We've done the same for HTML/JS/Others.

Reply:
Will do!

Reply:
Done. I kinda don't love the `SHOUTY_CASE` for constants, but hey.